### ISSUE TITLE
Getting "outside valid range for the datatype INTEGER” on insert when auto increment primary key is large BIGINT

### ISSUE THREAD

**Issue by DougEverly** [jeremyevans/sequel#1018]
```ruby
sql = "INSERT INTO table (pk, name) VALUES (NULL, 'Doug')”
pk = DB[sql].insert
```

When the auto-incrementing pk is "148171983513" BigInt for example.

The row gets inserted correctly, but seems the primary key return value is expected to be Int, not possible BigInt.

```
Sequel::DatabaseError: Java::ComMysqlJdbcExceptionsJdbc4::MySQLDataException: '1.48171983512E11' in column '1' is outside valid range for the datatype INTEGER.
       raise_error at /home/deverly/my_project/vendor/cache/jruby/2.2.0/gems/sequel-4.23.0/lib/sequel/database/misc.rb:423
         statement at /home/deverly/my_project/vendor/cache/jruby/2.2.0/gems/sequel-4.23.0/lib/sequel/adapters/jdbc.rb:653
  block in execute at /home/deverly/my_project/vendor/cache/jruby/2.2.0/gems/sequel-4.23.0/lib/sequel/adapters/jdbc.rb:246
              hold at /home/deverly/my_project/vendor/cache/jruby/2.2.0/gems/sequel-4.23.0/lib/sequel/connection_pool/threaded.rb:103
       synchronize at /home/deverly/my_project/vendor/cache/jruby/2.2.0/gems/sequel-4.23.0/lib/sequel/database/connecting.rb:255
           execute at /home/deverly/my_project/vendor/cache/jruby/2.2.0/gems/sequel-4.23.0/lib/sequel/adapters/jdbc.rb:245
    execute_insert at /home/deverly/my_project/vendor/cache/jruby/2.2.0/gems/sequel-4.23.0/lib/sequel/adapters/jdbc.rb:281
    execute_insert at /home/deverly/my_project/vendor/cache/jruby/2.2.0/gems/sequel-4.23.0/lib/sequel/dataset/actions.rb:937
            insert at /home/deverly/my_project/vendor/cache/jruby/2.2.0/gems/sequel-4.23.0/lib/sequel/dataset/actions.rb:336
```

**Comment by DougEverly** [comment]
I changed `rs.getInt(1)` to `rs.getLong(1)` in `lib/sequel/adapters/jdbc/mysql.rb` `#last_insert_id` and it seems to work for BigInt at least. Didn’t test with Int. Not sure if should/can check type or not on the `rs` ResultSet and conditionally do getInt or getLong.

**Comment by jeremyevans** [comment]
Fixed by the merging of #1019
